x-pack/plugins/painless_lab/public/application/components/request_flyout.tsx

```diff
@@ -40,7 +40,7 @@ export const RequestFlyout: FunctionComponent<Props> = ({
           <EuiFlexItem>
             {/* We need an extra div to get out of flex grow */}
             <div>
-              <EuiTitle size="m">
+              <EuiTitle size="m" data-test-subj="flyoutHeader">
```

Comment:
`question`: Do we feel safe checking header or it is better to validate `EuiFlyoutBody` is present and not empty at least?

Reply:
I think in order to validate that the flyout is open, it's good enough to just check the title. Checking the content of the flyout is planned for a follow-up anyway.

However, I'd suggest to make the test subject a bit more specific, e.g. `painlessLabRequestFlyoutHeader`.